Modules/vc-module-orders/VirtoCommerce.OrdersModule.Web/Controllers/Api/OrderModuleController.cs

```diff
@@ -110,8 +115,9 @@ public async Task<ActionResult<CustomerOrder>> GetByNumber(string number, [FromR
         {
             var searchCriteria = AbstractTypeFactory<CustomerOrderSearchCriteria>.TryCreateInstance();
             searchCriteria.Number = number;
-            //ToDo
-            //searchCriteria.ResponseGroup = OrderReadPricesPermission.ApplyResponseGroupFiltering(_securityService.GetUserPermissions(User.Identity.Name), respGroup); ;
+
+            var user = await _userManager.FindByNameAsync(User.Identity.Name);
```

Comment:
Could you make `GetUserPermissions()` method with this code, as it is used in multiple places?